### PR TITLE
Ensure no mutable state can affect my tests

### PR DESCRIPTION
**Pull request by ReDetection**
thanks @kevich

### CODE REVIEW

File: test/ConverterTest.php [hunk 1]
```diff
@@ -8,16 +8,20 @@ public function testRemovesColorProfileInformation()
     {
         $bin = self::converter()->convert(file_get_contents(__DIR__ . '/data/colorProfile.jpeg'), self::command('strip'));
         $tmpFile = self::tmpDir() . 'profilesCheckRemove.jpg';
+        @unlink(filename);
         file_put_contents($tmpFile, $bin);
         $this->assertEquals('', exec('identify -verbose "' . $tmpFile . '" | grep "Profile-"'));
+        @unlink(filename);
```
Comment (Magomogo) [inline test/ConverterTest.php]: What is the meaning of `filename`?

Comment (Magomogo) [reply test/ConverterTest.php]: @kevich

Comment (ReDetection) [reply test/ConverterTest.php]: okay, that should be `$tmpFile`

Comment (ReDetection) [reply test/ConverterTest.php]: @Magomogo please see #14

Comment (kevich) [reply test/ConverterTest.php]: My bad

**Pull request merged**